### PR TITLE
added lib.dmon.get / lib.dmon.bump and refactored dmon loop

### PR DESCRIPTION
**Pull request by invisig0th**

### CODE REVIEW

File: synapse/cortex.py [hunk 1]
```diff
@@ -3278,6 +3294,52 @@ async def addStormDmon(self, ddef):
...
+        if ddef is None:
+            return False
+
+        curv = ddef.get('enabled')
+
+        ddef['enabled'] = True
+        await self.stormdmonhive.set(iden, ddef)
+
+        if self.isactive and not curv:
+            dmon = self.stormdmons.getDmon(iden)
```
Comment (vEpiphyte) [inline synapse/cortex.py:3322]: Shouldn't this be a start() call to the dmon manager ?

Comment (vEpiphyte) [reply synapse/cortex.py:3322]: for the specific dmon iden?

File: synapse/tests/test_lib_storm.py [hunk 2]
```diff
@@ -226,6 +226,39 @@ async def test_lib_storm_basics(self):
 
             self.eq({'foo': 'bar'}, await core.callStorm('return($lib.dict(    foo    =    bar   ))'))
 
+            ddef0 = await core.callStorm('return($lib.dmon.add(${ $lib.queue.gen(hehedmon).put(lolz) $lib.time.sleep(10) }, name=hehedmon))')
```
Comment (vEpiphyte) [inline synapse/tests/test_lib_storm.py:229]: this feels like we're just throwing more and more and more stuff into this particular unit test

Comment (invisig0th) [reply synapse/tests/test_lib_storm.py:229]: yep!  i am.  to keep the cortex instantiation count down for cluster tests.

**Review by vEpiphyte — APPROVED** [review]

**Pull request merged**